CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Changelog for NeoFS Node
 - Validate storage node configuration before node startup (#1805)
 - `neofs-node -check` command to check the configuration file (#1805)
 - `flush-cache` control service command to flush write-cache (#1806)
+- Validate policy before container creation, add `--force` option to skip validation (#1704)
```

Comment:
I'd place `add --force` statement to `Updating from...` section, what do u think?

Reply:
Done